### ISSUE TITLE
Can I add application by this web UI to Fdroid repository ?

### ISSUE THREAD

**Issue by rwoloszyn** [dervishe-/f-droid-web#36]
As in topic ?

**Comment by dervishe-** [comment]
Nop, for the moment it's just a set of consultation pages. In near future, i plan to add an admin part which allow registered people to add apps and regenerate the several files. If you want to, you're welcome to participate. :-)

**Comment by rwoloszyn** [comment]
Thanks for answer.  I will take a look into it and maybe post some patches. From the other hand maybe you can explain me how repository metadata is refreshed here.  Do you just base on already prepared metadata by fdroid server tools?

**Comment by dervishe-** [comment]
Exactly, every time i add an app to the repository, i do it manually with fdroid update -c after erasing the index.xml and index.jar located in the website root.
The main goal was to make just a simple visualization of the repository's content.

**Comment by rwoloszyn** [comment]
OK. Thanks, so additional job to call fdroid update -c and erasing files which you mentioned will be needed. I'll think about it in my spare time.

**Comment by dervishe-** [comment]
Yes, but it's not the only step to do, we need in this case a web form in order to collect several information about the app like license type, categories, sources location, etc...

**Comment by rwoloszyn** [comment]
Yep I get that. Some web form  + kind of job (worker) which will call update function. Question is do you have any good one for PHP which you prefer ? Celery ?
